### PR TITLE
TRUNK-5775:  Upgrade org.dbunit

### PR DESCRIPTION
**Pull request by achilep**
## Description of what I changed

The idea is upgrade org.dbunit dependencie to the latest version .
This imply this change : 
org.dbunit:dbunit:2.6.0 --> 2.7.0

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5775

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -496,7 +496,7 @@
 			<dependency>
 				<groupId>org.dbunit</groupId>
 				<artifactId>dbunit</artifactId>
-				<version>2.6.0</version>
+				<version>2.7.0</version>
```
Comment (HerbertYiga) [inline pom.xml:499]: try out the changes with reference application

Comment (achilep) [reply pom.xml:499]: thank @HerbertYiga I tested it in the same way as all the other PRs that I perform, by executing the jetty: run command, with the legacyUI module install.

Comment (achilep) [reply pom.xml:499]: hello @dkayiwa @mozzy11 @sherrif10 @HerbertYiga  

I tested with the standalone reference application 
Screenshot from 2020-06-16 11-01-27

**Comment by sherrif10** [comment]
Thanks @dkayiwa , i think @achilep is now relified

**Pull request merged**